### DISCUSSION TITLE
Cadence of version releases?

### DISCUSSION THREAD

**Discussion by alfonsosn** [agencyenterprise/react-native-health#137] [General]
Hello @macelai and @lucaspbordignon!

Been loving the effort that you two are doing on the library. Great work! 

I had a question, do you all have a specific cadence for when you create releases/tags? We recently authored a commit and we are thinking of using it in order to fully migrate to the library. While it's not urgent, I was just curious how you all scope timelines for your releases. This will help us better asses our own product roadmaps as we complete this migration.

Thanks again!

**Comment by macelai** [comment]
Hey @alfonsosn

Thank you! Loving to build it with the community! We don't have a specific cadence to create a release, but if you need a new release for something specific, just let us know and we will do our best to achieve it.

**Reply by alfonsosn** [reply]
Thanks for the answer @macelai. Always a pleasure working w you all!

**Comment by alfonsosn** [comment]
Perhaps when both: [PR 135](https://github.com/agencyenterprise/react-native-health/pull/135) and [PR 108](https://github.com/agencyenterprise/react-native-health/pull/108) are merged. We can create a new release version?

**Reply by macelai** [reply]
If I don't hear from them by the end of the week, we can create a new release

**Reply by alfonsosn** [reply]
Awesome, thanks @macelai !

**Reply by lucaspbordignon** [reply]
Just a follow-up on the thread, we've just released [V1.10](https://github.com/agencyenterprise/react-native-health/releases/tag/v1.10.0)

As @macelai mentioned, we don't have a regular schedule of releases. We definitely could work on that, but given the uncertainty of PRs and changes in the repo, we haven't agreed on any process yet 🙂
